fuse/kenner_pd.py

```diff
@@ -28,20 +28,28 @@ def fuse_events(pprr):
             ]
         },
     }, ['uid'])
+    builder.extract_events(uof, {
+        events.UOF_INCIDENT: {
+            'prefix': 'incident',
+        },
+    }, ['uid', 'uof_uid'])
     return builder.to_frame()
 
 
 if __name__ == '__main__':
     post_event = pd.read_csv(data_file_path(
         'match/post_event_kenner_pd_2020.csv'))
     pprr = pd.read_csv(data_file_path('clean/pprr_kenner_pd_2020.csv'))
+    uof = pd.read_csv(data_file_path('clean/uof_kenner_pd_2005_2021.csv'))
+    uof_df = rearrange_use_of_force(uof)
     events_df = rearrange_event_columns(pd.concat([
         post_event,
-        fuse_events(pprr)
+        fuse_events(pprr, uof)
     ]))
     ensure_uid_unique(events_df, 'event_uid', True)
-    ensure_data_dir('fuse')
     rearrange_personnel_columns(pprr).to_csv(data_file_path(
```

Comment:
Since uof data does contain unmatched officers, you might want to fuse personnel from uof as well: `fuse_personnel(pprr, uof)`

Reply:
Yes thank you